VAMobile/src/screens/PaymentsScreen/PaymentHistory/NoPayments/NoPaymentsScreen.tsx

```diff
@@ -13,7 +13,7 @@ function NoPaymentsScreen() {
 
   return (
     <VAScrollView>
-      <AlertBox border="informational">
+      <AlertWithHaptics variant="info">
         <TextView variant="MobileBodyBold" accessibilityLabel={a11yLabelVA(t('payments.noPayments.title'))}>
           {t('payments.noPayments.title')}
```

Comment:
Could be partially simplified moving two of these to the `header` and `description` props to AlertWithHaptics.